packages/webpack-cli/package.json

```diff
@@ -51,9 +51,6 @@
     "@webpack-cli/init": {
       "optional": true
     },
-    "@webpack-cli/migrate": {
-      "optional": true
-    },
```

Comment:
Keep it, otherwise it will be breaking change

Reply:
added